core_lib/src/structure/layerbitmap.cpp

```diff
@@ -44,6 +44,18 @@ BitmapImage* LayerBitmap::getLastBitmapImageAtFrame(int frameNumber, int increme
     return static_cast<BitmapImage*>(getLastKeyFrameAtPosition(frameNumber + increment));
 }
 
+void LayerBitmap::replaceLastBitmapAtFrame(const BitmapImage* replaceWithImage)
```

Comment:
Looks like we can add a general `replaceLastAtFrame()` in the base class for all types of layers?